src/main/java/studio/magemonkey/fabled/data/sql/tables/FabledPlayersSQL.java

```diff
@@ -0,0 +1,180 @@
+package studio.magemonkey.fabled.data.sql.tables;
+
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Player;
+import studio.magemonkey.codex.mccore.config.Filter;
+import studio.magemonkey.codex.mccore.config.parse.DataSection;
+import studio.magemonkey.codex.mccore.config.parse.YAMLParser;
+import studio.magemonkey.fabled.Fabled;
+import studio.magemonkey.fabled.PlayerLoader;
+import studio.magemonkey.fabled.api.player.PlayerAccounts;
+import studio.magemonkey.fabled.data.io.IOManager;
+import studio.magemonkey.fabled.data.sql.SQLManager;
+import studio.magemonkey.fabled.data.sql.SQLUtils;
+
+import java.io.*;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+public class FabledPlayersSQL extends IOManager {
+
+    private final String Table = "Fabled_players";
+
+    public FabledPlayersSQL() {
+        super(Fabled.inst());
+        try (PreparedStatement create = SQLManager.connection()
+                .prepareStatement("CREATE TABLE IF NOT EXISTS " + Table + "("
+                        + "UUID varchar(36), "
+                        + "Data MEDIUMTEXT)")) {
+            create.execute();
+        } catch (SQLException e) {
+            Fabled.inst().getLogger().warning("[SQL:FusionPlayersSQL:FusionPlayersSQL] Something went wrong with the sql-connection: "
+                            + e.getMessage());
+        }
+    }
+
+    public PlayerAccounts loadPlayerAccounts(OfflinePlayer player) {
+        if (player == null || player.getName() == null) return null;
+        try(Connection connection = SQLManager.connection()) {
+            PreparedStatement statement = connection.prepareStatement("SELECT * FROM " + Table + " WHERE UUID = ?");
+            statement.setString(1, player.getUniqueId().toString());
+            ResultSet result = statement.executeQuery();
+            if(result.next()) {
+                return SQLUtils.load(player, result.getString("Data"));
+            }
+
+        } catch (SQLException e) {
+            Fabled.inst().getLogger().warning("[SQL:FusionPlayersSQL:loadPlayersAccounts] Something went wrong with the sql-connection: "
+                            + e.getMessage());
+        }
+
+        return null;
+    }
+
+    public Map<UUID, PlayerAccounts> loadAllOnlinePlayerAccounts() {
+        Map<UUID, PlayerAccounts> result = new HashMap<>();
+        for (Player player : Bukkit.getOnlinePlayers()) {
+            result.put(player.getUniqueId(), loadPlayerAccounts(player));
```

Comment:
This would be more performant if we could get all online players at once instead of in consecutive queries. We'd have to create an overload `public List<PlayerAccounts> loadPlayerAccounts(Collection<Player> players)` in order to do it. Then we could leverage `select * from " + table + " where uuid in (LIST)`

Reply:
Thats actually a good practise I should refer on in future. You are right. Thanks for pointing that out!